Audio tests are failing

Compiling from current master, the following tests fail:
```
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running org.eclipse.smarthome.core.audio.test.AudioManagerTest
Tests run: 23, Failures: 3, Errors: 0, Skipped: 0, Time elapsed: 30.853 sec <<< FAILURE! - in org.eclipse.smarthome.core.audio.test.AudioManagerTest
audio manager plays stream from mp3 audio files(org.eclipse.smarthome.core.audio.test.AudioManagerTest)  Time elapsed: 0.134 sec
audio manager plays mp3 audio files(org.eclipse.smarthome.core.audio.test.AudioManagerTest)  Time elapsed: 0.004 sec
sink is registered(org.eclipse.smarthome.core.audio.test.AudioManagerTest)  Time elapsed: 0.009 sec
audio manager sets the volume of not registered sink to zero(org.eclipse.smarthome.core.audio.test.AudioManagerTest)  Time elapsed: 0.013 sec
the volume of a null sink is zero(org.eclipse.smarthome.core.audio.test.AudioManagerTest)  Time elapsed: 0 sec
source is added in parameter options(org.eclipse.smarthome.core.audio.test.AudioManagerTest)  Time elapsed: 0.033 sec
setVolume method handles IOException(org.eclipse.smarthome.core.audio.test.AudioManagerTest)  Time elapsed: 0.005 sec
audio manager does not process streams if there is no registered sink(org.eclipse.smarthome.core.audio.test.AudioManagerTest)  Time elapsed: 10.137 sec  <<< FAILURE!
java.lang.AssertionError: Could not find AudioServlet
Expected: is not null
     but: was null
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.junit.Assert.assertThat(Assert.java:865)
	at org.junit.Assert$assertThat$0.callStatic(Unknown Source)
	at org.eclipse.smarthome.core.audio.test.AudioOSGiTest$_initializeAudioServlet_closure4.doCall(AudioOSGiTest.groovy:136)
	at org.eclipse.smarthome.core.audio.test.AudioOSGiTest$_initializeAudioServlet_closure4.doCall(AudioOSGiTest.groovy)
	at sun.reflect.GeneratedMethodAccessor8.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:324)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:278)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1016)
	at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:39)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:112)
	at org.eclipse.smarthome.test.OSGiTest.waitForAssert(OSGiTest.groovy:218)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:207)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:56)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:49)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:153)
	at org.eclipse.smarthome.test.OSGiTest.waitForAssert(OSGiTest.groovy:189)
	at org.eclipse.smarthome.test.OSGiTest.waitForAssert(OSGiTest.groovy)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:207)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:56)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:49)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:133)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:141)
	at org.eclipse.smarthome.core.audio.test.AudioOSGiTest.initializeAudioServlet(AudioOSGiTest.groovy:132)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:207)
	at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:56)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallCurrent(CallSiteArray.java:49)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:133)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:137)
	at org.eclipse.smarthome.core.audio.test.AudioManagerTest.assertServedStream(AudioManagerTest.groovy:338)
```

```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] Eclipse SmartHome .................................. SUCCESS [  0.179 s]
[INFO] Eclipse SmartHome Bundles .......................... SUCCESS [  0.010 s]
[INFO] Eclipse SmartHome Test Components .................. SUCCESS [  0.007 s]
[INFO] Eclipse SmartHome Core Components .................. SUCCESS [  0.006 s]
[INFO] Eclipse SmartHome Core ............................. SUCCESS [  1.909 s]
[INFO] Eclipse SmartHome Configurations ................... SUCCESS [  0.007 s]
[INFO] Eclipse SmartHome Configuration Core ............... SUCCESS [  0.363 s]
[INFO] Eclipse SmartHome Configuration XML ................ SUCCESS [  0.191 s]
[INFO] Eclipse SmartHome AutoUpdate Binding ............... SUCCESS [  0.906 s]
[INFO] Eclipse SmartHome Test ............................. SUCCESS [  0.716 s]
[INFO] Eclipse SmartHome Auth Components .................. SUCCESS [  0.007 s]
[INFO] Eclipse SmartHome JAAS Authentication .............. SUCCESS [  0.076 s]
[INFO] Eclipse SmartHome Configuration Dispatcher ......... SUCCESS [  0.107 s]
[INFO] Eclipse SmartHome Configuration Dispatcher Tests ... SUCCESS [ 11.067 s]
[INFO] Eclipse SmartHome Configuration Core Tests ......... SUCCESS [  2.990 s]
[INFO] Eclipse SmartHome I/O Components ................... SUCCESS [  0.010 s]
[INFO] Eclipse SmartHome Console .......................... SUCCESS [  0.104 s]
[INFO] Eclipse SmartHome Thing ............................ SUCCESS [  0.548 s]
[INFO] Eclipse SmartHome Configuration Discovery .......... SUCCESS [  0.224 s]
[INFO] Eclipse SmartHome Configuration Discovery Tests .... SUCCESS [ 15.297 s]
[INFO] Eclipse SmartHome Config XML Tests ................. SUCCESS [  4.514 s]
[INFO] Eclipse SmartHome Core Audio ....................... SUCCESS [  0.125 s]
[INFO] Eclipse SmartHome Core Binding XML ................. SUCCESS [  0.126 s]
[INFO] Eclipse SmartHome Core Thing XML ................... SUCCESS [  0.173 s]
[INFO] Eclipse Smarthome Audio Test ....................... FAILURE [01:54 min]
[INFO] Eclipse SmartHome Sample Extension Service ......... SKIPPED
[INFO] Eclipse SmartHome REST Interface ................... SKIPPED
[INFO] Eclipse SmartHome Core ID .......................... SKIPPED
[INFO] Eclipse SmartHome ID Tests ......................... SKIPPED
[INFO] Eclipse SmartHome Core Tests ....................... SKIPPED
[INFO] Eclipse SmartHome Thing Tests ...................... SKIPPED
```

--- a/bundles/core/org.eclipse.smarthome.core.audio.test/AudioOSGiTest.launch
+++ b/bundles/core/org.eclipse.smarthome.core.audio.test/AudioOSGiTest.launch
@@ -31,7 +31,7 @@
 <stringAttribute key="org.eclipse.jdt.launching.PROGRAM_ARGUMENTS" value="-os ${target.os} -ws ${target.ws} -arch ${target.arch} -nl ${target.nl} -consoleLog"/>
 <stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="org.eclipse.smarthome.core.audio.test"/>
 <stringAttribute key="org.eclipse.jdt.launching.SOURCE_PATH_PROVIDER" value="org.eclipse.pde.ui.workbenchClasspathProvider"/>
-<stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value="-Dorg.osgi.service.http.port=9090 -Djetty.home.bundle=org.eclipse.jetty.osgi.boot"/>
+<stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value="-Djetty.port=9090 -Djetty.home.bundle=org.eclipse.jetty.osgi.boot"/>
 <stringAttribute key="pde.version" value="3.3"/>
 <stringAttribute key="product" value=""/>
 <booleanAttribute key="run_in_ui_thread" value="true"/>

--- a/bundles/core/org.eclipse.smarthome.core.audio.test/pom.xml
+++ b/bundles/core/org.eclipse.smarthome.core.audio.test/pom.xml
@@ -163,7 +163,7 @@
               <autoStart>true</autoStart>
             </bundle>
           </bundleStartLevel>
-          <argLine>-Dorg.osgi.service.http.port=9090 -Djetty.home.bundle=org.eclipse.jetty.osgi.boot</argLine>
+          <argLine>-Djetty.port=9090 -Djetty.home.bundle=org.eclipse.jetty.osgi.boot</argLine>
         </configuration>
       </plugin>
     </plugins>

--- a/bundles/core/org.eclipse.smarthome.core.audio.test/src/test/groovy/org/eclipse/smarthome/core/audio/test/AudioOSGiTest.groovy
+++ b/bundles/core/org.eclipse.smarthome.core.audio.test/src/test/groovy/org/eclipse/smarthome/core/audio/test/AudioOSGiTest.groovy
@@ -11,9 +11,6 @@ package org.eclipse.smarthome.core.audio.test
 import static org.hamcrest.CoreMatchers.*
 import static org.junit.Assert.*
 
-import java.util.Locale;
-
-import org.apache.commons.lang.SystemUtils
 import org.eclipse.smarthome.config.core.ConfigConstants
 import org.eclipse.smarthome.core.audio.*
 import org.eclipse.smarthome.core.audio.internal.AudioManagerImpl
@@ -38,12 +35,12 @@ class AudioOSGiTest extends OSGiTest {
     protected AudioSink audioSinkFake
     protected AudioStream audioStream
     protected AudioServlet audioServlet
-    
+
     protected def audioSourceMock
 
     protected final String AUDIO_SERVLET_PROTOCOL = "http"
     protected final String AUDIO_SERVLET_HOSTNAME = "localhost"
-    protected final int AUDIO_SERVLET_PORT = 8080
+    protected final int AUDIO_SERVLET_PORT = 9090
 
     protected byte[] testByteArray = [0, 1, 2]
 

